app/views/modals/loginDialog.html

```diff
@@ -12,12 +12,12 @@ <h3 ng-if="status401" translate>Authorization Required</h3>
       Please sign in with one of the following services:
     </p>
     <div class="row">
-      <div class="col-sm-5 col-sm-offset-1">
+      <div ng-if="facebookLogin" class="{{relayLogin ? 'col-sm-5 col-sm-offset-1' : 'col-sm-6 col-sm-offset-3' }}">
```

Comment:
Are you using the correct `class` here? As in `<div ng-if="facebookLogin" class="{{relayLogin ? '` @emiliano1

Reply:
@hlbraddock Yes, the class is correct.
If the Facebook button appears alone, the class will be `col-sm-6 col-sm-offset-3`.
If it appears together with the Relay button, the class will be `col-sm-5 col-sm-offset-1` (as it was before).